api/server/router/network/network_routes.go

```diff
@@ -153,7 +153,7 @@ func (n *networkRouter) postNetworkDisconnect(ctx context.Context, w http.Respon
 		return errors.NewRequestForbiddenError(err)
 	}
 
-	return n.backend.DisconnectContainerFromNetwork(disconnect.Container, nw, disconnect.Force)
+	return n.backend.DisconnectContainerFromNetwork(disconnect.Container, vars["id"], disconnect.Force)
```

Comment:
I didnt do that because, its being removed in https://github.com/docker/docker/pull/25962 :)